Modules/IxNetRestApiPortMgmt.py

```diff
@@ -240,8 +186,13 @@ def linkUpDown(self, port, action='down'):
             port: <list>: A list of ports in a list.  [[ixChassisIp, str(card), str(port)]] -> ['10.10.10.1', '1', '3']
             action: <str>: up|down
         """
-        vport = self.ixnObj.getVports([port])[0]
-        self.ixnObj.post(self.ixnObj.sessionUrl+'/vport/operations/linkUpDn', data={'arg1': [vport], 'arg2': action})
+
+        action = action.lower()
+        self.ixnObj.logInfo("Link {} operation for ports {}".format(action,port))
+        for eachPort in port:
+            self.ixnObj.logInfo("\n Make port {} to {} state".format(":".join(eachPort),action))
+            for vport in self.ixNetObj.Vport.find(AssignedTo=":".join(eachPort)):
```

Comment:
Here "for" loop not required use like below.
self.ixNetObj.Vport.find(AssignedTo=":".join(eachPort))